### PR TITLE
add support for dotnet build

### PR DESCRIPTION
**Pull request by SimonCropp**
fixes https://github.com/Particular/PlatformDevelopment/issues/1458

### CODE REVIEW

File: src/nuget.config [hunk 1]
```diff
@@ -8,6 +8,7 @@
   <packageSources>
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
     <add key="particular myget.org" value="https://www.myget.org/F/particular/" />
+    <add key="Local Integration Source" value="..\nugets" />
```
Comment (tmasternak) [inline src/nuget.config]: Is that for local development with unreleased packages?

Comment (SimonCropp) [reply src/nuget.config]: yes. u have an integration solution for smoke testing the full msbuild pipeline

**Comment by SimonCropp** [comment]
@Particular/metrics-maintainers what is your preference here. do u want this as part of the v2 release or hold off?

**Comment by SimonCropp** [comment]
any objection to me smoke testing, merging and promoting a new beta to nuget?

**Comment by dvdstelt** [comment]
Seems fine by me, I'd so go ahead.

**Pull request merged**